FirebaseMLModelDownloader/Tests/Integration/ModelDownloaderIntegrationTests.swift

```diff
@@ -12,568 +12,577 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import XCTest
-@testable import FirebaseCore
-@testable import FirebaseInstallations
-@testable import FirebaseMLModelDownloader
-
-extension UserDefaults {
-  /// Returns a new cleared instance of user defaults.
-  static func createTestInstance(testName: String) -> UserDefaults {
-    let suiteName = "com.google.firebase.ml.test.\(testName)"
-    let defaults = UserDefaults(suiteName: suiteName)!
-    defaults.removePersistentDomain(forName: suiteName)
-    return defaults
-  }
-
-  /// Returns the existing user defaults instance.
-  static func getTestInstance(testName: String) -> UserDefaults {
-    let suiteName = "com.google.firebase.ml.test.\(testName)"
-    return UserDefaults(suiteName: suiteName)!
-  }
-}
-
-final class ModelDownloaderIntegrationTests: XCTestCase {
-  override class func setUp() {
-    super.setUp()
-    // TODO: Use FirebaseApp internal for test app.
-    let bundle = Bundle(for: self)
-    if let plistPath = bundle.path(forResource: "GoogleService-Info", ofType: "plist"),
-       let options = FirebaseOptions(contentsOfFile: plistPath) {
-      FirebaseApp.configure(options: options)
-    } else {
-      XCTFail("Could not locate GoogleService-Info.plist.")
+// Skip keychain tests on Catalyst and macOS. Tests are skipped because they
```

Comment:
@paulb777, there is a handy _Hide whitespace_ option. It makes the diff on this PR manageable.

<img width="624" alt="Screen Shot 2022-08-28 at 12 02 23 AM" src="https://user-images.githubusercontent.com/36927374/187056806-77e12224-afd9-440a-9d34-8407ed2a4622.png">

Reply:
Resolving so I can merge for nightlies